{"levelname": "INFO", "asctime": "2022-12-13T00:22:12.013Z", "name": "root", "message": "Unable to find a row with primary key 6c8b58ff-0c35-45e2-9f0b-3dc154cd37b9, in Collection table.", "lineno": 43, "pathname": "/single-cell-data-portal/backend/common/entities/entity.py"}
